src/main/scala/com/github/sbt/sbom/BomExtractor.scala

```diff
@@ -95,22 +119,42 @@ class BomExtractor(settings: BomExtractorParams, report: UpdateReport, log: Logg
       component
     }
 
+    private def artifactPaths(moduleReport: ModuleReport): Seq[File] =
+      moduleReport.artifacts
+        .map { case (_, file) =>
+          file
+        }
+        .filter { file =>
+          file.exists() && file.isFile
+        }
+
+    private def hashes(files: Seq[File]): Seq[Hash] =
+      files.flatMap { file =>
+        BomUtils.calculateHashes(file, settings.schemaVersion).asScala
+      }
+
     private def licenseChoice: Option[LicenseChoice] = {
-      val licenses: Seq[model.License] = moduleReport.licenses.map { case (name, mayBeUrl) =>
-        model.License(name, mayBeUrl)
+      val licenses: Seq[License] = moduleReport.licenses.map { case (name, urlOption) =>
+        val license = new License()
+        urlOption.foreach { licenseUrl =>
+          /*TODO: LicensesArchive.bundled.findByNormalizedUrl(licenseUrl).foreach { archiveLicense =>
+            license.setId(archiveLicense.id)
+          }*/
```

Comment:
now I see why #94 :)

what are typical values for `name` here? would that be useful to distinguish between e.g. `GPL-2.0` and `GPL-2.0-or-later`?